src/keycloak/urls_patterns.py

```diff
@@ -148,6 +148,11 @@
 URL_ADMIN_CLIENT_SCOPE = URL_ADMIN_CLIENT_SCOPES + "/{scope-id}"
 URL_ADMIN_CLIENT_SCOPES_ADD_MAPPER = URL_ADMIN_CLIENT_SCOPE + "/protocol-mappers/models"
 URL_ADMIN_CLIENT_SCOPES_MAPPERS = URL_ADMIN_CLIENT_SCOPES_ADD_MAPPER + "/{protocol-mapper-id}"
+URL_ADMIN_CLIENT_SCOPE_ROLE_MAPPINGS = URL_ADMIN_CLIENT_SCOPE + "/scope-mappings"
+URL_ADMIN_CLIENT_SCOPE_ROLE_MAPPINGS_REALM = URL_ADMIN_CLIENT_SCOPE_ROLE_MAPPINGS + "/realm"
+URL_ADMIN_CLIENT_SCOPE_ROLE_MAPPINGS_CLIENT = (
+    URL_ADMIN_CLIENT_SCOPE_ROLE_MAPPINGS + "/clients/{client}"
```

Comment:
For consistency sake, I'd suggest to name the parameter `client-id` instead of just `client`, to be explicit that this is the unique ID, not something else